src/state/slices/opportunitiesSlice/resolvers/osmosis/index.ts

```diff
@@ -28,37 +29,41 @@ export const osmosisLpOpportunitiesMetadataResolver = async ({
 
   const lpOpportunitiesById: Record<LpId, OpportunityMetadata> = {}
 
-  const liquidityPools = await getPools()
+  const { OsmosisLP } = selectFeatureFlags(state)
 
-  for (const pool of liquidityPools) {
-    const toAssetIdParts: ToAssetIdArgs = {
-      assetNamespace: 'ibc',
-      assetReference: `gamm/pool/${pool.id}`,
-      chainId: osmosisChainId,
-    }
+  if (OsmosisLP) {
```

Comment:
Prefer early return or even throw (which will be handled gracefully with how we `Promise.allSettled` thunks) but this is fine